OsmIntegrator/Interfaces/UseCaseResponse.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using System.Net;
+
+namespace OsmIntegrator.Interfaces {
+    public abstract class AUseCaseResponse
+    {        
+        public string Message { get; }
+        public HttpStatusCode StatusCode { get; }
```

Comment:
W net core jak zwracasz takie odpowiedzi jak:
Ok();
NotFound();
BadRequest();
To od razu są statusy uwzględnione. Czy ten dodatkowy status jest potrzebny?

Reply:
To zaszłość. Wywaliłem ją